packages/core/scripts/gas-rebate/index.js

```diff
@@ -69,33 +100,21 @@ async function parseRevealEvents(committedVotes, revealedVotes, priceData, multi
         web3.eth.getBlock(latestCommitEvent.blockNumber),
         web3.eth.getTransactionReceipt(latestCommitEvent.transactionHash)
       ]);
-      const commitGasUsed = parseInt(commitReceipt.gasUsed, 16);
-
-      const key = `${voter}-${roundId}-${identifier}-${requestTime}`;
-      const val = {
-        voter,
-        roundId,
-        identifier,
-        requestTime,
-        reveal: {
-          transactionBlock: transactionBlock.number,
-          hash: transactionReceipt.transactionHash,
-          gasUsed
-        },
-        commit: {
-          transactionBlock: commitBlock.number,
-          hash: commitReceipt.transactionHash,
-          gasUsed: commitGasUsed
-        }
+      val.commit = {
+        transactionBlock: commitBlock.number,
+        hash: commitReceipt.transactionHash,
+        gasUsed: parseInt(commitReceipt.gasUsed),
+        txnTimestamp: commitBlock.timestamp
       };
-
-      revealVotersToRebate[key] = val;
-      progressBarReveal.update(i + 1);
     } else {
-      throw new Error(
-        `Could not find VoteCommitted event matching the reveal event: ${JSON.stringify(reveal.returnValues)}`
-      );
+      // console.error(
```

Comment:
I'm leaving this commented out because printing logs in the middle of a CLI-Progress makes it look bad